src/course/syllabus/final-project/schedule.md

```diff
@@ -0,0 +1,7 @@
+# Final Project Schedules
```

Comment:
Is this file actually linked to from anywhere?

Reply:
It is auto-linked under the Schedule tab for design and each build week. I added this to resolve the 404 when clicking that tab.